hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/codec/DecodingFacade.java

```diff
@@ -156,7 +163,7 @@ public static AccountID convertLeftPaddedAddressToAccountId(
             @NonNull final UnaryOperator<byte[]> aliasResolver,
             @NonNull final Predicate<AccountID> exists) {
         var accountID = convertLeftPaddedAddressToAccountId(leftPaddedAddress, aliasResolver);
-        if (!exists.test(accountID)) {
+        if (!exists.test(accountID) && !accountID.hasAlias()) {
```

Comment:
Given above, I think we should delete this until it becomes possible for `aliasResolver` to return a non-mirror address. 🤔